### DISCUSSION TITLE
docker compose -f docker-compose-CN.yml up -d not success

### DISCUSSION THREAD

**Discussion by comet3000** [infiniflow/ragflow#2410] [Q&A]
hi all,

(base) harvest@harvest-virtual-machine:/ragflow/docker$ sudo docker compose -f docker-compose-CN.yml up -d
WARN[0000] The "MACOS" variable is not set. Defaulting to a blank string. 
[+] Running 60/1
[+] Running 62/1⣿⣿⣿⣿⣷⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿] 8.459GB / 11.34GB Pulling             ⠙ ragflow [⣿⣿ ✔ ragflow Pulled                                                                                                     2035.3s 
[+] Running 4/5
 ✔ Container ragflow-redis   Running                                                                                     0.0s 
 ✔ Container ragflow-es-01   Healthy                                                                                     2.4s 
 ✔ Container ragflow-mysql   Healthy                                                                                     2.4s 
 ✔ Container ragflow-minio   Running                                                                                     0.0s 
 ⠋ Container ragflow-server  Starting                                                                                    3.5s 
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/ragflow/docker/nginx/nginx.conf" to rootfs at "/etc/nginx/nginx.conf": mount /ragflow/docker/nginx/nginx.conf:/etc/nginx/nginx.conf (via /proc/self/fd/6), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

what`s wrong this it?

**Comment by KevinHuSh** [comment]
Have you changed something? What about changing it back?
